packages/webpack-config/web-default/custom-service-worker.js

```diff
@@ -0,0 +1,78 @@
+/* global self */
+
+/**
+ * Store notification icon string in service worker.
+ * Ref: https://stackoverflow.com/a/35729334/2603230
+ */
+self.addEventListener('message', function(event) {
```

Comment:
```suggestion
self.addEventListener('message', event => {
```
for consistency with your other listeners